tests/acceptance/blocks/filtersearchcomponent.js

```diff
@@ -23,6 +23,6 @@ export default class FilterSearchComponentBlock {
       .typeText(this._input, displayValue)
       .pressKey('space')
       .wait(500)
-      .click(this._selector.find('.js-yext-autocomplete-option').withExactText(displayValue));
```

Comment:
How come the text from clicking the option does not match the option's exact display value?

Reply:
think it was just a spurious failure, our current usages should always be able to match exactly. I changed it from `withText` to `withExactText` in my PR right before this

Reply:
if this pops up frequently we can try bumping the selector timeout, since this was a headless acceptance failure we don't have the browserstack video to see what happened during this run